source/frontend/src/components/maps/leaflet/Control/Search/PropertyQuickInfoContainer.tsx

```diff
@@ -0,0 +1,154 @@
+import { geoJSON } from 'leaflet';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaEye, FaSearchPlus } from 'react-icons/fa';
+import styled from 'styled-components';
+
+import { useMapStateMachine } from '@/components/common/mapFSM/MapStateMachineContext';
+import { SectionField } from '@/components/common/Section/SectionField';
+import TooltipWrapper from '@/components/common/TooltipWrapper';
+import usePathGenerator from '@/features/mapSideBar/shared/sidebarPathGenerator';
+import { useLtsa } from '@/hooks/useLtsa';
+import { exists, firstOrNull, isValidString, pidFormatter } from '@/utils';
+import { formatNames } from '@/utils/personUtils';
+
+export const PropertyQuickInfoContainer: React.FC<React.PropsWithChildren> = () => {
+  const [ownerNames, setOwnerNames] = useState('');
+  const { requestFlyToBounds, requestFlyToLocation, mapLocationFeatureDataset } =
+    useMapStateMachine();
+
+  const pathGenerator = usePathGenerator();
+
+  const locationInfo = useMemo(
+    () => firstOrNull(mapLocationFeatureDataset?.parcelFeatures)?.properties,
+    [mapLocationFeatureDataset?.parcelFeatures],
+  );
+
+  const { ltsaRequestWrapper } = useLtsa();
+
+  const getLtsaExecute = ltsaRequestWrapper.execute;
+
+  const hasMultipleProperties = useMemo(
+    () => mapLocationFeatureDataset?.parcelFeatures?.length > 1,
+    [mapLocationFeatureDataset?.parcelFeatures?.length],
+  );
+
+  const getOwnerInfo = useCallback(
+    async (pid: string) => {
+      if (isValidString(pid)) {
+        const ltsaOrders = await getLtsaExecute(pid);
+        console.log(ltsaOrders.titleOrders);
```

Comment:
a couple console.logs left in this PR.

Reply:
technically you could have ismounted checks here, but not a pattern we've been following consistently.